src/apps/dashboard/dashboard-notification-list/dashboard-notification-list.tsx

```diff
@@ -322,11 +324,20 @@ const DashboardNotificationList: FC<DashboardNotificationListProps> = ({
         />
       )}
 
-      <MaterialDetailsModal modalId={`${loanDetails}${modalLoanDetailsId}`}>
+      <MaterialDetailsModal
+        modalId={constructMaterialDetailsModalId(
```

Comment:
Maybe we should declare modalId as a const to avoid multiple function calls and duplicate code? 😊